resize2fs: add page (#4296)
tldr-pages/tldr: pages/linux/resize2fs.md

# resize2fs

> > Resize an ext2, ext3 or ext4 file system.
> > Does not resize the underlying partition.
> > File system must be unmounted.

Reviewer: Perhaps this will read better?
```suggestion
> Resize an ext2, ext3 or ext4 filesystem.
> Does not resize the underlying partition, and the filesystem must be unmounted.
```

This way we can avoid 3 lines of description.

> - Automatically resize file system:

Reviewer: Perhaps this will read better?
```suggestion
- Automatically resize a filesystem:
```

Also, `file system` → `filesystem` I think, based on other pages (e.g. `mkfs`).

Author: File system seems to be the most common usage:
https://books.google.com/ngrams/graph?content=filesystem%2Cfile+system&corpus=26&year_start=1940&year_end=2019&smoothing=3&direct_url=t1%3B%2Cfilesystem%3B%2Cc0%3B.t1%3B%2Cfile%20system%3B%2Cc0

Author: Although manpages seem to prefer filesystem, and this repo has more cases of filesystem over file system.

`resize2fs {{/dev/sdXN}}`

> - Resize the file system to 40G with a percentage completion bar:

Reviewer: Perhaps this will read better?
```suggestion
- Resize the filesystem to a size of 40G, displaying a progress bar:
```

`resize2fs -p {{/dev/sdXN}} 40G`

> - Shrink the file system to its minimum possible size:

Reviewer: Perhaps this will read better?
```suggestion
- Shrink the filesystem to its minimum possible size:
```
